books/free-programming-books-de.md

```diff
@@ -75,12 +76,14 @@
 ### <a id="csharp"></a>C\#
 
 * [Einführung in das Programmieren mit C#](https://www.uni-trier.de/universitaet/wichtige-anlaufstellen/zimk/downloads-broschueren/programmierung/c) - Bernhard Baltes-Götz (PDF)
+* [FUNDAMENTALS OF COMPUTER PROGRAMMING WITH C#](https://introprogramming.info/wp-content/uploads/2013/07/Books/CSharpEn/Fundamentals-of-Computer-Programming-with-CSharp-Nakov-eBook-v2013.pdf) (PDF)
 * [Programmieren in C#: Einführung](http://www.highscore.de/csharp/einfuehrung)
 * [Visual C# 2012](http://openbook.rheinwerk-verlag.de/visual_csharp_2012) - Andreas Kühnel (Online)
 
 
 ### <a id="cpp"></a>C++
 
+* [A Complete Guide to Programming in C++](https://www.idpoisson.fr/volkov/C++.pdf) (PDF)
```

Comment:
This appears not to be an authorized distribution. Please see docs/CONTRIBUTING.md guidelines.